packages/shared/src/components/brief/BriefListItem.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { Fragment } from 'react';
+import type { MouseEvent, ReactElement, ReactNode } from 'react';
+import classNames from 'classnames';
+import Link from 'next/link';
+import { BriefGradientIcon } from '../icons/BriefGradient';
+import {
+  Typography,
+  TypographyColor,
+  TypographyTag,
+  TypographyType,
+} from '../typography/Typography';
+import type { PillProps } from '../Pill';
+import { Pill } from '../Pill';
+import { IconSize } from '../Icon';
+import { LockIcon } from '../icons';
+import { webappUrl } from '../../lib/constants';
+
+export type BriefListItemProps = {
+  briefId: string;
+  className?: string;
+  title: ReactNode;
+  pill?: Omit<PillProps, 'className'>;
+  readTime?: string;
+  postsCount?: number;
+  sourcesCount?: number;
+  isRead?: boolean;
+  isLocked?: boolean;
+  onClick?: (briefId: string, event: MouseEvent<HTMLAnchorElement>) => void;
+};
+
+export const BriefListItem = ({
+  briefId,
+  className,
+  title,
+  pill,
+  readTime,
+  postsCount,
+  sourcesCount,
+  isRead,
+  isLocked,
+  onClick,
+}: BriefListItemProps): ReactElement => {
+  return (
+    <Link
+      href={`${webappUrl}posts/${briefId}`}
+      onClick={(event) => {
+        if (typeof onClick === 'function') {
+          onClick(briefId, event);
+        }
+      }}
+    >
+      <article
+        className={classNames(
+          'flex w-full items-center gap-4 rounded-16 border border-border-subtlest-tertiary p-3',
+          className,
+        )}
+      >
+        <div className="flex items-center">
+          <BriefGradientIcon secondary={!isRead} size={IconSize.Size48} />
+        </div>
+        <div className="flex flex-col gap-1">
+          <div className="flex items-center gap-2">
+            <Typography
+              type={TypographyType.Title3}
+              bold
+              color={
+                isRead ? TypographyColor.Quaternary : TypographyColor.Primary
+              }
+            >
+              {title}
+            </Typography>
+            {!!pill && (
+              <Pill
+                {...pill}
+                className="invert !self-auto bg-accent-bacon-default py-0.5 text-text-primary"
+              />
```

Comment:
Why not we just turn this into ReactNode?

Reply:
That should be it 🙏

Reply:
I mean, instead of receiving `pill` props, we can just send a ReactNode that will do the rendering of the Pill itself. Anyway, not a blocker, maybe there are benefits to this too, like ensuring the Pill is rendered as intended.

Reply:
Ah got it, I'll look into it when I continue implementation.